src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputLineLengthLongImportStatements.java

```diff
@@ -9,8 +9,6 @@
 
 package com.puppycrawl.tools.checkstyle.checks.sizes.linelength;
 
-import com.puppycrawl.tools.checkstyle.grammar.comments.InputFullOfSinglelineComments;
-import static com.puppycrawl.tools.checkstyle.grammar.comments.InputFullOfSinglelineComments.main;
 
```

Comment:
Name of file claims that long imports are main target of this test.
Can we put some other long import here that is available in jdk.
Can we put extra spaces after `.` ? To make short import to be long.

Reply:
spaces are allowed in imports.
prove:
```
✔ ~/java/github/romani/checkstyle [master|✚ 1⚑ 1] 
$ git diff
diff --git a/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java b/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java
index d3288dd..5180046 100644
--- a/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java
@@ -19,7 +19,7 @@
 
 package com.puppycrawl.tools.checkstyle.internal;
 
-import static com.google.common.truth.Truth.assertWithMessage;
+import static com.google.common.  truth.Truth.assertWithMessage;
 import static java.lang.Integer.parseInt;
 
 import java.beans.PropertyDescriptor;
```

and this compiles without problems.

so use any import from jdk and put spaces in it to be long.

Reply:
Okay Thanks for the help will add the spaces then